chip8_vm/README.md

```diff
@@ -0,0 +1,37 @@
+Virtual Machine for the CHIP-8 programming language
+==
+This crate implements a virtual machine of the CHIP-8 programming language. It
+can be used as the back end for CHIP-8 emulator projects / debuggers etc.
+
+Status
+==
+* All 35 original Chip-8 instructions are implemented.
+
+Usage
+==
+This crate only has no system library dependencies, only dependencies on other
+rust crates. It can be used as a back end for your own emulator by adding it as
+a dependency in your `Cargo.toml`.
+
+To depend on the latest released version check [crates.io] (https://crates.io/crates/chip8_vm)
```

Comment:
Cargo allows version wildcards, so in theory we could have `0.x.y` version crates and tell users to use
```toml
[dependencies]
chip8_vm = "0.*"
```
This would mean they get the latest matching version whenever they update (`cargo update --package chip8_vm`).

Once Rust itself is somewhat stable, we could switch to `1.x.y` versions and tell users to use a specific version (and rely on `SemVer` compatibility).